project/src/test/java/com/google/sps/data/PlacesTest.java

```diff
@@ -14,38 +14,23 @@
 
 package com.google.sps.data;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableList;
 import com.google.maps.model.LatLng;
 
 @RunWith(JUnit4.class)
 public final class PlacesTest {
 
   /** A valid Place object with name "name1". */
-  private static final Place PLACE_1 =
-      Place.builder()
-          .setName("name1")
-          .setWebsiteUrl("website@google.com")
-          .setPhone("+97250-0000-000")
-          .setRating(4)
-          .setPriceLevel(3)
-          .setLocation(new LatLng(35.35, 30.30))
-          .build();
+  private static final Place PLACE_1 = getValidPlaceBuilder().setName("name1").build();
```

Comment:
If you already have a utility method that retrieves a builder, use it directly in the test (without constants). 

Notice that test readers will look at PLACE_1 -> go to definition -> go to the method (and only then realize what's going on).

Notice that using constants can be helpful only when you're not hiding important test details from the reader.

Reply:
We do that even if it's used in a few different functions? We don't mind here about code repetition?

Reply:
You mean calling the method with the same argument in different tests? If that's it then yes, we don't mind it (if it makes sense).
Generally speaking in tests we prefer test readability over everything (and sometimes it does entail code repetition to a point, but that's where one might think what is irrelevant information to a test (and can be extracted to a method / constant) and what should remain there.

Reply:
No, that's not what I meant. I meant PLACE_1 and PLACE_2 are used in 3 different functions - so they will have the same code if it's not in an outer constant, but if that's preferred, then sure, done